src/python/ASO/Rucio/Transfer.py

```diff
@@ -192,6 +206,7 @@ def readContainerRuleID(self):
                                  f' Transfer Container rule ID: {self.containerRuleID}'\
                                  f' Publish Container rule ID: {self.publishRuleID}')
         except FileNotFoundError:
+            self.containerRuleID = ''
```

Comment:
sorry, I am lost.. there are so many containers now that I am not sure that self.containerRuleId is a useful name. For sure at this moment I do not know what it refers to. Maybe add a description at the top lines 48-57 ?

Reply:
I want to enforce what I wrote in https://github.com/dmwm/CRABServer/pull/8019/files#diff-21e172d79b282189754bb0c7903111655ac95e23f103fe2cd3cdfbe719135ce6R48-R50

I should not mix this fix in this PR. Sorry.

Reply:
`containerRuleId` is rule ID of Transfer Container.

I agree we need to change the name. However not in this PR.

Reply:
fine. Thanks